doc/ref_arch/openstack/chapters/chapter04.rst

```diff
@@ -536,10 +536,9 @@ Example Host Configurations
 
 Let us refer to the data traffic networking configuration of
 :numref:`Basic Profile Host Configuration` to be part of the hp-B1-a and
-hp-B4-a host profiles and this requires the configurations as Table
-`Configuration of Basic Flavor Capabilities`_.
+hp-B4-a host profiles and this requires the configurations as
+:numref:`Configuration of Basic Flavor Capabilities`.
```

Comment:
@pgoyal01 Adding :numref: is in conflit with PR #3165 where numref are removed to ease conversion.

Reply:
@karinesevilla Then we need to make changes in other chapters. For example, Chapter 3 has 4 ":numref:":
- :numref:`OpenStack Core Services`; line 307
- :numref:`OpenStack Services Topology` line 356
- :numref:`OpenStack Services Topology`, line 368
- :numref:`Network Fabric - Physical` line 676

I will run through other chapters and identify the numref's and then make the necessary changes.

Reply:
@pgoyal01 It is already done (part of PR#3165)

Reply:
@karinesevilla Yes -- I saw that and so will not progress. Thanks.

THis current PR will have to be rebase by @collivier

Reply:
@pgoyal01 yes I can rebase it. pending